content/pt/docs/concepts/cluster-administration/cluster-administration-overview.md

```diff
@@ -0,0 +1,76 @@
+---
+reviewers:
+- jcjesus
+- femrtnz
```

Comment:
I believe those are the content reviewers, not the translation reviewers. Maybe we should keep both here.

Reply:
I fixed. Tks Felipe.